src/outputs.ts

```diff
@@ -54,6 +20,8 @@ export interface BulkUploadReport {
  * @returns (BulkRedirectListItem[]) Raw redirect list entries for a CF Bulk Redirect List
  */
 export const makeBulkList = (input: RedirectProps[]): BulkRedirectListItem[] => {
+  const default_locale = DEFAULT_LOCALE || 'en-US'; // assume en-US
```

Comment:
Added support for settable DEFAULT_LOCALE in the wrangler.toml file. Falls back to en-US if not set.